templates/views/blog/_post.html.twig

```diff
@@ -0,0 +1,23 @@
+<div class="row posts">
+    {% for article in paginator %}
```

Comment:
`paginator` is not defined anymore, how does this work ?

Reply:
Hello! I fixed this and all the multiple times the posts appear on the view. Now, only the last 3 posts are displayed on the index page only. And I fixed the paginator issue on the view with the controller, now it works. 

The thing is : I created a new template called _post.html.twig to avoid overweighting the views. So, I included it on blog.html.twig, index.html.twig. and since paginator is called on the blog action in controller (and displays a certain number of posts per page - especially the latest posts), I had to define it on the controller. I still have to fix the paginator variable on the view to make it work too.